addons/website_slides/controllers/main.py

```diff
@@ -165,7 +165,8 @@ def _get_channel_progress(self, channel, include_quiz=False):
         if not request.env.user._is_public() and channel.is_member:
             slide_partners = request.env['slide.slide.partner'].sudo().search([
                 ('channel_id', '=', channel.id),
-                ('partner_id', '=', request.env.user.partner_id.id)
+                ('partner_id', '=', request.env.user.partner_id.id),
+                ('slide_id.active', '=', True)
```

Comment:
Should be slide_id, in, slides.ids to be coherent with above search

I noticed we have your fix in master but it does not seem correct to me.

Reply:
Hagrid

Reply:
@tde-banana-odoo _Not guilty, Your Honour!_ : https://github.com/odoo/odoo/commit/38b5a666c50d47a0efe060c031d68a2d70689dc6

(I did not notice this fix in master, master is not for tech support :p )

Reply:
Well by your fix it was more "the same fix", not that you were the author, indeed ^^ .

Anyway slides.ids is better than checking active.

Reply:
Done